src/css/controls/imports/autostartmute.less

```diff
@@ -0,0 +1,23 @@
+@import "../../shared-imports/vars";
+@import "../../shared-imports/icons";
```

Comment:
What happens in 7.10 with errors & controls false?

We're going to need to use the error player state when controls is false, because there is no display container without controls.